File: goraceloty-front-end/src/app/entity/Hotel.ts
export class Hotel {
  hotelID: number;
  hotelName: string;
  standard: number;
  country: string;
  city: string;
  childrenAllowed: boolean;
  address: string;

  constructor(
    hotelID: number,
    hotelName: string,
    standard: number,
    country: string,
    city: string,
    childrenAllowed: boolean,
    address: string
  ) {
    this.hotelID = hotelID;
    this.hotelName = hotelName;
    this.standard = standard;
    this.country = country;
    this.city = city;
    this.childrenAllowed = childrenAllowed;
    this.address = address;
  }
}

File: api-gateway/src/main/java/com/goraceloty/apigateway/hotels/entity/Hotel.java
package com.goraceloty.apigateway.hotels.entity;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class Hotel {
    public Long hotelID;
    private String hotelName;
    private Integer standard;
    private String country;
    private String city;
    private Boolean childrenAllowed;
    private String address;
}
